new_platforms/docs/GettingStarted.md

```diff
@@ -44,6 +44,12 @@ This reference platform runs OP-TEE to support ARM TrustZone TAs. For more detai
 * [Building an Open Enclave TrustZone TA on Linux](linux_arm_dev.md)
 * [Running the Sample Client on the Grapeboard](sample_sockets.md#building-for-grapeboard)
 
+## Linux Host Application and ARM TrustZone Emulation
+
+QEMU provides an emulated ARM TrustZone environment to debug your REE and TEE code that faithfully reproduces the dynamics of real ARM TrustZone-capable hardware. You can use GDB or any GDB-compatible GUI to debug your code.
```

Comment:
suggest spliting into two lines in the source

Reply:
Good catch, thanks. i didn't realize using GitHub's integrated editor. Fixed.